pages/alignment-files/[id].js

```diff
@@ -50,42 +57,53 @@ export default function AlignmentFile({
           <FileHeaderDownload file={alignmentFile} />
         </ObjectPageHeader>
         <JsonDisplay item={alignmentFile} isJsonFormat={isJson}>
+          <FileDataItems item={alignmentFile} fileSet={fileSet}></FileDataItems>
+          <DataAreaTitle>Alignment Details</DataAreaTitle>
           <DataPanel>
             <DataArea>
-              <FileDataItems
-                item={alignmentFile}
-                fileSet={fileSet}
-                derivedFrom={derivedFrom}
-              >
-                {referenceFiles.length > 0 && (
-                  <>
-                    <DataItemLabel>Reference Files</DataItemLabel>
-                    <DataItemValue>
-                      <SeparatedList>
-                        {referenceFiles.map((file) => (
-                          <Link href={file["@id"]} key={file["@id"]}>
-                            {file.accession}
-                          </Link>
-                        ))}
-                      </SeparatedList>
-                    </DataItemValue>
-                  </>
-                )}
+              {referenceFiles.length > 0 && (
                 <>
-                  <DataItemLabel>Redacted</DataItemLabel>
+                  <DataItemLabel>Reference Files</DataItemLabel>
                   <DataItemValue>
-                    {alignmentFile.redacted ? "Yes" : "No"}
+                    <SeparatedList>
+                      {referenceFiles.map((file) => (
+                        <Link href={file["@id"]} key={file["@id"]}>
+                          {file.accession}
+                        </Link>
+                      ))}
+                    </SeparatedList>
                   </DataItemValue>
                 </>
-                <DataItemLabel>Filtered</DataItemLabel>
+              )}
+              <>
+                <DataItemLabel>Redacted</DataItemLabel>
                 <DataItemValue>
-                  {alignmentFile.filtered ? "Yes" : "No"}
+                  {alignmentFile.redacted ? "Yes" : "No"}
                 </DataItemValue>
-                <DataItemLabel>Content Summary</DataItemLabel>
-                <DataItemValue>{alignmentFile.content_summary}</DataItemValue>
-              </FileDataItems>
+              </>
+              <DataItemLabel>Filtered</DataItemLabel>
+              <DataItemValue>
+                {alignmentFile.filtered ? "Yes" : "No"}
+              </DataItemValue>
             </DataArea>
           </DataPanel>
+          {derivedFrom?.length > 0 && (
```

Comment:
We know `derivedFrom` contains either an array of objects or an empty array, so you can just do `derivedFrom.length > 0` and so forth — we know `derivedFrom` exists, even if it contains no elements.

Reply:
changed